README.md

```diff
@@ -1,24 +1,69 @@
-# README
+# school-library-app
+The Blog app will be a classic example of a blog website. You will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Built With
 
-Things you may want to cover:
+- Technologies used:
+  > Ruby
+  > Gemfile
+  > IRB
+  > Rails
+  > PostgreSql
+  > Rspec
 
-* Ruby version
 
-* System dependencies
+## Getting Started
 
-* Configuration
+### Prerequisites
 
-* Database creation
+You need to have a browser and this tools installed:
 
-* Database initialization
+- Git
+- NodeJS
```

Comment:
- [ ] [OPTIONAL] I think you don't need NodeJS to run your Rails app. Kindly remove this as well.